libs/sdk-backend-tiger/src/backend/workspace/insights/mocks/visualizationClasses.ts

```diff
@@ -172,19 +172,18 @@ export const visualizationClasses: IVisualizationClass[] = sortBy(
                 uri: "/gdc/md/gtl83h4doozbp26q0kf5qg8uiyu4glyn/obj/1000",
             },
         },
-        // geochart must be disabled for now on tiger backend due to missing mapbox token infrastructure there (RAIL-3058)
```

Comment:
Commented code is equal to uncommented code on line 175-186. Should it be commented or not?

Reply:
sry rebase artifact, will remove it